packages/core/styles/CHANGELOG.md

```
@@ -2,11 +2,24 @@
 
 Nimbus Styles deprive all styles needed to build nimbus components.
 
-## 2026-02-03 `9.55.0`
+## 2026-02-04 `9.55.0`
+
+#### 🎨 Visual changes
```

markdownlint-cli2 (0.20.0)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)